src/GitHub.Api/OutputProcessors/VersionOutputProcessor.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+
+namespace GitHub.Unity
+{
+    class VersionOutputProcessor : BaseOutputProcessor<Version>
+    {
+        public override void LineReceived(string line)
+        {
+            if (String.IsNullOrEmpty(line))
+                return;
+
+            var gitVersion = "git version ";
+            if (line.StartsWith(gitVersion))
+            {
+                line = line.Substring(gitVersion.Length);
+                var strings = line.Split(new[] { "." }, StringSplitOptions.None);
+
+                RaiseOnEntry(new Version(Int32.Parse(strings[0]), Int32.Parse(strings[1]), Int32.Parse(strings[2])));
```

Comment:
I am pretty sure someone has already written a regex somewhere to parse the git version, but in any case something like `/git version ([\d]+)\.([\d]+)\.([\d]+)` should probably work and be resilient enough?

Reply:
Changed to a regex